aten/src/ATen/core/Macros.h

```diff
@@ -29,3 +32,18 @@
 #define AT_DISABLE_COPY_AND_ASSIGN(classname) \
   classname(const classname&) = delete;       \
   classname& operator=(const classname&) = delete
+
+namespace at {
+#if defined(__ANDROID__)
+
+inline int stoi(const std::string& str) {
+  std::stringstream ss;
+  int n = 0;
+  ss << str;
+  ss >> n;
+  return n;
+}
+#else
+using std::stoi;
```

Comment:
this is not ideal, because now this function has different arguments depending on if you are on android or not.  Instead, how about just changing the body?

Reply:
This is what caffe2 currently do: https://github.com/pytorch/pytorch/blob/master/caffe2/core/common.h

do you mean
```
namespace at {
inline int stoi(const std::string& str) {
#if defined(__ANDROID__)
  std::stringstream ss;
  int n = 0;
  ss << str;
  ss >> n;
  return n;
#else
 return std::stoi(str);
#endif
}
} // namespace at
```
?

Then we'll probably need to change callsite to use at::stoi? is that OK?

Reply:
that sounds good to me, what do you think?

Reply:
yeah, we can do that.